### DISCUSSION TITLE
Tidy Filesystem

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [AsyncBanana/microdiff#28] [Ideas]
Have you thought about sorting the source files into a `Source`
folder and the assets such as your logo into `Resources`?

Also, how about adding a `.editorconfig`?

**Comment by AsyncBanana** [comment]
I have considered sorting files into more specific folders but found it unnecessary because there aren't many files overall. However, I might change my mind later. As for adding a `.editorconfig`, I will probably just stick with using Prettier for formatting.
